examples/audiocapture.py

```diff
@@ -26,10 +22,9 @@
 pg.init()
 
 # init_subsystem(INIT_AUDIO)
-names = [get_audio_device_name(x, 1) for x in range(get_num_audio_devices(1))]
+names = get_audio_device_names(True)
```

Comment:
The keyword arg here would make it readable. Which makes me wonder if a separate function for capture devices would be better? Because then people would use it rather than supply True/1 without the keyword.

Reply:
I was thinking about that too, but I wanted to keep the PR small, just making sure the audio device system works with the new function from a previous PR.

The new function is a lot nicer because it constructs the list, and it doesn't return byte strings.

Maybe `get_playback_device_names()` and `get_capture_device_names()`? It probably shouldn't be just `get_capture_devices` because that suggests it returns the AudioDevice itself, rather than just the name.